src/Components/Switcher/index.tsx

```diff
@@ -54,9 +57,9 @@ const Switcher: React.FC<Props> = (props: Props) => {
         }}
       />
       {
-        label && <label htmlFor={id}>{label}</label>
+        <span>{label ? label : ''}</span>
```

Comment:
Are you sure, that we need to have a span block, if we do not have label

Reply:
Yes. This span - is a whole switcher tho